Library/Homebrew/rubocops/lines_cop.rb

```diff
@@ -67,7 +67,71 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
             next unless block_arg.source.size>1
             problem "\"inreplace <filenames> do |s|\" is preferred over \"|#{block_arg.source}|\"."
           end
+
+          [:rebuild, :version_scheme].each do |method_name|
+            find_method_with_args(body_node, method_name, 0) do
+              problem "'#{method_name} 0' should be removed"
+            end
+          end
+
+          [:mac?, :linux?].each do |method_name|
+            next unless formula_tap == "homebrew-core"
+            find_instance_method_call(body_node, "OS", method_name) do |check|
+              problem "Don't use #{check.source}; Homebrew/core only supports macOS"
+            end
+          end
+
+          find_method_with_args(body_node, :fails_with, :llvm) do
+            problem "'fails_with :llvm' is now a no-op so should be removed"
+          end
+
+          find_method_with_args(body_node, :system, /^(otool|install_name_tool|lipo)$/) do
+            next if @formula_name == "Cctools"
+            problem "Use ruby-macho instead of calling #{@offensive_node.source}"
+          end
+
+          find_every_method_call_by_name(body_node, :system).each do |method_node|
+            next if @formula_name =~ /^Kibana(\@\d+(\.\d+)?)?$/
```

Comment:
Could we document why Kibana is given a pass on this check?

Reply:
Also, the previous regex used a lowercase K.

Reply:
Thanks! I was trying to get formula name from Class Name. The above regex is wrong, it supposed to be `/^Kibana(AT\d+)?$/ `
Original regex is lower because formula name is obtained from filename itself. I'll change this to extract formula's name from filename.